pymedphys/_trf/decode/header.py

```diff
@@ -33,7 +33,7 @@ def determine_header_length(trf_contents):
 def decode_header(trf_header_contents):
```

Comment:
needs docstring
and it would be helpful to have comments that discuss what the various regex pieces were expected to look like, perhaps with an example.
Or a reference to any trf file format documentation...

Reply:
Yup, fair regarding the docstring. Wasn't a part of this PR though so will address this separately.

Regarding the other questions this has been reverse engineered based on the data we have. TRF file format documentation would help, but unfortunately I don't have anything official on that front.